tests/test_utils_fix_filename.py

```diff
@@ -15,7 +16,10 @@ def test_one_dot():
 
 def test_two_dots():
     fname = "aa.a.a"
-    assert fix_filename(fname) == "{aa.a}.a"
+    if os.name == 'posix':
```

Comment:
Is it possible to test both, by manually changing `os.name`?
Something like this:
```python
original_os_name = os.name
try:
    os.name = 'posix'
    assert fix_filename(fname) == "{aa.a}.a"
    os.name = 'nt'
    assert fix_filename(fname) == "aa.a.a"
finally:
    os.name = original_os_name
```